code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -121,6 +121,8 @@
 		if(!just_sleeping)
 			if(HAS_TRAIT(src, TRAIT_SUICIDED))
 				. += span_warning("[t_He] appear[p_s()] to have committed suicide... there is no hope of recovery.")
+			if(HAS_TRAIT(src, TRAIT_PLAYING_MAFIA))
+				. += span_warning("[t_He] appear[p_s()] to still be lingering somewhere... [p_they()] will surely be back soon.")
```

Comment:
I decided to just not have any text so people don't confuse it for the suicide text.
I remember last time we changed them and it was reverted because people couldn't tell the difference.